pkgs/misc/lz4json/default.nix

```
@@ -0,0 +1,38 @@
+{ pkgs ? import <nixpkgs> { system = builtins.currentSystem; }
+, lib ? pkgs.lib
+, stdenv ? pkgs.stdenv
+, fetchFromGitHub ? pkgs.fetchFromGitHub
+}:
```

```suggestion
{ lib
, stdenv
, fetchFromGitHub
, lz4
, pkg-config
}:
```

Take only inputs that are needed to build this package.